src/Adapter/MSTest.CoreAdapter/Execution/TypeCache.cs

```diff
@@ -36,20 +36,20 @@ internal class TypeCache : MarshalByRefObject
         /// <summary>
         /// Helper for reflection API's.
         /// </summary>
-        private ReflectHelper reflectionHelper;
+        private readonly ReflectHelper reflectionHelper;
 
         /// <summary>
         /// Assembly info cache
         /// </summary>
-        private Dictionary<Assembly, TestAssemblyInfo> testAssemblyInfoCache;
+        private readonly Dictionary<Assembly, TestAssemblyInfo> testAssemblyInfoCache;
```

Comment:
Why does this have to be readonly?

Reply:
Latest analysis tool in VS is giving suggestions for fiels to be readonly if they are instantiated only in the declaration/constructor.

Reply:
in short, what @jayaranigarg mentioned. Since `testAsseblyInformation` field is only declared in TypeCache constructor, as suggested by the VS code analysis tool, it's recommended to have it `readonly`.